server.js

```diff
@@ -45,6 +45,14 @@ app.use(
   })
 );
 
+// No-index and No-follow in "not production"
+if (env.STAGING_SERVER) {
```

Comment:
let's turn this into a `const`, e.g.,

```js
const ON_STAGING_SERVER = env.STAGING_SERVER === `True`;
```

and use `ON_STAGING_SERVER` instead.